reports/report_csv.py

```diff
@@ -0,0 +1,174 @@
+    topic = [t for t in MAJOR_TOPICS if t[0] == row[1]][0][1]
+    writer.writerow({'Topic': topic, 'Answer': row[0], 'Count': counts_list[row]})
+
+def ws_41_csv(writer, counts_list, row, **kwargs):
+    all_topics = [y for x in TOPICS for y in x[1]]
+    topic = [x[1] for x in all_topics if x[0] == row[1]][0]
+    writer.writerow({'Topic': topic[4:].strip(), 'Answer': row[0], 'Count': counts_list[row]})
+
+def ws_47_csv(writer, counts_list, row, **kwargs):
+    topic = [t for t in MAJOR_TOPICS if t[0] == row[1]][0][1]
+    answer = 'Agree' if row[0] == 1 else 'Disagree'
+    writer.writerow({'Topic': topic, 'Answer': answer, 'Count': counts_list[row]})
+
+def ws_48_csv(writer, counts_list, row, **kwargs):
+    topic = [t for t in MAJOR_TOPICS if t[0] == row[1]][0][1]
+    answer = 'Agree' if row[0] == 1 else 'Disagree'
+    gender = get_gender(kwargs['gender'])
+    writer.writerow({'Topic': topic, 'Gender': gender, 'Answer': answer, 'Count': counts_list[row]})
+
+def ws_83_csv(writer, counts_list, row, **kwargs):
+    regions = kwargs['regions']
+    for gender, region in counts_list[row]:
+        count = counts_list[row][(gender, region)]
+        region = [x[1] for x in regions if x[0] == region][0]
+        gender = get_gender(gender)
+        writer.writerow({'Topic': row, 'Region': region, 'Gender': gender, 'Count': count})
+
+def ws_85_csv(writer, counts_list, row, **kwargs):
+    regions = kwargs['regions']
+    function, region = row
+    region = [x[1] for x in regions if x[0] == region][0]
+    function = [func[1] for func in FUNCTION if func[0] == function]
+    if function:
+        function = function[0][4:] # [4:] is used to remove the (number) part
+        writer.writerow({'Region': region, 'Function': function, 'Count': counts_list[row]})
+
+def ws_92_csv(writer, counts_list, row, **kwargs):
+    topic, agree_disagree = row
+    count = counts_list[row]
+    region = kwargs['region']
+    topic = [t for t in MAJOR_TOPICS if t[0] == topic][0][1]
+    agree_disagree = 'Agree' if agree_disagree == 1 else 'Disagree'
+    writer.writerow({'Region': region, 'Topic': topic, 'Answer': agree_disagree, 'Count': count})
+
+def ws_93_csv(writer, counts_list, row, **kwargs):
+    topic, yes_no = row
+    count = counts_list[row]
+    region = kwargs['region']
+    topic = [t for t in MAJOR_TOPICS if t[0] == topic][0][1]
+    yes_no = 'Yes' if yes_no == 'Y' else 'No'
+    writer.writerow({'Region': region, 'Topic': topic, 'Answer': yes_no, 'Count': count})
+
+def ws_97_csv(writer, counts_list, row, **kwargs):
+    regions = kwargs['regions']
+    for region_answer in counts_list[row]:
+        answer, region = region_answer
+        agree_disagree = 'Agree' if answer == 1 else 'Disagree'
+        region = [x[1] for x in regions if x[0] == region][0]
+        count = counts_list[row][region_answer]
+        writer.writerow({'Region': region, 'Topic': row, 'Agree/Disagree': agree_disagree, 'Count': count})
```

Comment:
Instead of `Agree/Disagree`, shouldn't we put the actual point of discussion here? e.g. `Stereotype` (This is applicable to the `Yes/No` as well.

Reply:
I'm not sure how this will appear on the graph. 
But I think it should be `Agree/Disagree` since the question(the actual point of discussion) will be what is displayed on the graph and we will be filtering by `Agree/Disagree`

Reply:
From the [documentation](https://app.gitbook.com/@openup/s/wazimap-ng/data-admin/creating-datasets) this should be like the column name: `Geography`, `Age`, `Race`, `Gender`, etc. `Yes/No` or `Agree/Disagree` don't fit but `GenderEquality`, `Stereotypes`, etc. do.

... but yes, if not sure, we can run as is first time and make changes after seeing how it looks visually.